src/action_server/actions/find.py

```diff
@@ -21,8 +23,7 @@ class Find(Action):
     def __init__(self):
         Action.__init__(self)
         # TODO: change this to a python dictionary schema
-        self._required_field_prompts = {'object': " What exactly would you like me to find? ",
-                                        'location': " Where should I look? "}
+        self._required_field_prompts = {'object': " What exactly would you like me to find? "}
         self._required_skills = ['head', 'base', 'rightArm', 'speech']
 
     def _point_at_person(self, person):
```

Comment:
I think you'll do yourself a favour when splitting this function up a bit

Reply:
The functionality is now working. I will refactor the individual actions in separate dedicated branches.